src/braket/experimental/autoqasm/README.md

```diff
@@ -68,7 +68,7 @@ def bell_state() -> None:
     cnot(0, 1)
 ```
 
-You can view the output format, which is OpenQASM, by running `bell_state().to_ir()`.
+You can view the output format, which is OpenQASM, by running `bell_state.to_ir()`.
```

Comment:
Yes, though if a user only wants to view, they will get a nicer version with `bell_state.display()`